libcxx/utils/generate_feature_test_macro_components.py

```diff
@@ -1236,7 +1236,8 @@ def add_version_header(tc):
         },
         {
             "name": "__cpp_lib_three_way_comparison",
-            "values": {"c++20": 201907},
+            "values": {"c++20": 201711},
+            # {"c++20": 201711} # P1614R2 The Mothership has Landed (see P1902R1 Missing feature-test macros 2017-2019)
             "headers": ["compare"],
             "unimplemented": True,
```

Comment:
Shouldn't this be marked as implemented? I think I am misunderstanding something.

Reply:
`strong_order` for `long double` is not yet supported ([Godbolt link](https://godbolt.org/z/dxhMM6cza)).

Reply:
Yes it should. After changing I formatted the file and  it seems the file was not formatted before so got a lot of unrelated changes. Seems I forgot to add the hunk that removed this line.

Indeed `long double` is not entirely complete; When NaNs are used the ordering is not available. (`long double` on some platforms has padding bits, which we can't handle.)

This was pointed out in the bug report and I didn't get strong objections to set the FTM. IMO `long double` is rather rare so I don't feel the omission is a huge issue. Still it would be great if we could enable this.

(As pointed out above P1614R2 did not actually bump the FTM; it was fixed later.)